cluster/src/main/scala/io/snappydata/metrics/SnappyMetricsClass.scala

```diff
@@ -0,0 +1,207 @@
+/*
+ * Changes for SnappyData data platform.
+ *
+ * Portions Copyright (c) 2017-2019 TIBCO Software Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+package io.snappydata.metrics
+
+import com.pivotal.gemfirexd.internal.engine.Misc
+import com.pivotal.gemfirexd.internal.engine.ui.{ClusterStatistics, MemberStatistics, SnappyExternalTableStats, SnappyRegionStats}
+import io.snappydata.SnappyTableStatsProviderService
+import java.util
+import org.apache.spark.SparkContext
+import org.apache.spark.groupon.metrics.UserMetricsSystem
+import org.apache.spark.groupon.metrics._
+
+import scala.collection.mutable
+
+object SnappyMetricsClass {
+
+  val oldSizeMap = collection.mutable.Map.empty[String, Int]
+
+  val region = Misc.getMemStore.getMetadataCmdRgn
+
+  def init(sc: SparkContext): Unit = {
+
+    // initialize metric system with cluster id as metrics namespace
+    val clusterUuid = region.get("__ClusterID__")
+    UserMetricsSystem.initialize(sc, clusterUuid)
+
+    // store every member diskStore ID to metadataCmdRgn
+    putMembersDiskStoreIdInRegion()
+
+    val memberEntries = region.getAll(region.keySet())
```

Comment:
Its name should be all meta entries or something as at his point you are collecting everything.

Reply:
Added suggested change